index.html

```diff
@@ -3,6 +3,7 @@
 
 <head>
   <meta charset="UTF-8">
+  <meta name="google-site-verification" content="MAGtB2GHGYz6jBJi37SzGuOeIaKWw0QolEgWWQ6uChI" />
```

Comment:
Just curious, how do you get the hash?

Reply:
It is provided by Google. When you add a site to Google Search Console, Google will provide this code to you and ask you to put that on your site to verify the ownership.

Reply:
got it.